sdk/containerregistry/azure-containers-containerregistry/src/main/java/com/azure/containers/containerregistry/authentication/package-info.java

```diff
@@ -0,0 +1,7 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+/**
+ * This package contains classes for the container registry project.
+ */
+package com.azure.containers.containerregistry.authentication;
```

Comment:
I don't see any classes in this package. Is this package-info required?

Reply:
Missed this. I moved all the files from here to the implementation package.

---
In reply to: [597948739](https://github.com/Azure/azure-sdk-for-java/pull/19926#discussion_r597948739) [](ancestors = 597948739)